### ISSUE TITLE
IE9.js stops IE8 using @font-face fonts

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#229]
```
Without IE9.js, my IE8 in Vista renders type specified via @font-face
declaration.

With IE9.js, it initially renders in correct font as expected, but then
changes it to Arial (presumably once the Javascript is executed).
```

Original issue reported on code.google.com by `davidoli...@googlemail.com` on 3 Mar 2010 at 4:53

**Comment by GoogleCodeExporter** [comment]
```
Test case please!
```

Original comment by `dean.edw...@gmail.com` on 3 Mar 2010 at 5:33
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
I can confirm the bug as reported by 'brighteyesdavidoliver'. Here is the test 
case:

http://www.vanner.sk/testcases/ie9js_ie8_fontface/testcase.html

thank you for reviewing
```

Original comment by `Jaroslav...@gmail.com` on 9 Apr 2010 at 2:20

**Comment by GoogleCodeExporter** [comment]
```
Sorry I never got round to providing a test case. Manic!

Thanks Jaroslav.
```

Original comment by `davidoli...@googlemail.com` on 9 Apr 2010 at 2:23

**Comment by GoogleCodeExporter** [comment]
```
The same for Windows 7 x64
```

Original comment by `Jaroslav...@gmail.com` on 9 Apr 2010 at 2:39

**Comment by GoogleCodeExporter** [comment]
Original comment by `dean.edw...@gmail.com` on 9 Apr 2010 at 4:33
- Changed state: **Re-open**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Please repair fast! Aka. Bump.
```

Original comment by `stf.alx....@gmail.com` on 23 Apr 2010 at 7:23

**Comment by GoogleCodeExporter** [comment]
```
Ha ha. OK. I will spend some time on this project over the weekend. I'll try to 
get rid 
of the regressions too.
```

Original comment by `dean.edw...@gmail.com` on 23 Apr 2010 at 9:07

**Comment by GoogleCodeExporter** [comment]
```
Hello Dean, 

this bug is really annoying because it breaks already implemented functionality 
in still quite stupid IE8. 
This library is otherwise the savior if one need to start implement the CSS3 
selectors right now, which is 
what many people starts to do. But in starting era of web fonts embedding it 
renders useless, which is 
pitty.

thank you for re-considering to make this bug the top priority for repairing 
```

Original comment by `Jaroslav...@gmail.com` on 24 Apr 2010 at 6:11

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
@Dean, here it is. Basicly it is enough to send the .eot files but I include
everything, just for sure.

thank you Dean
```

Original comment by `Jaroslav...@gmail.com` on 25 Apr 2010 at 3:33

Attachments:
- [ie9js_ie8_fontface.zip](https://storage.googleapis.com/google-code-attachments/ie7-js/issue-229/comment-11/ie9js_ie8_fontface.zip)

**Comment by GoogleCodeExporter** [comment]
```
You'll be pleased to know that I have fixed this bug now. Will check in shortly.
```

Original comment by `dean.edw...@gmail.com` on 25 Apr 2010 at 3:35

**Comment by GoogleCodeExporter** [comment]
Original comment by `dean.edw...@gmail.com` on 25 Apr 2010 at 3:36
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 2.1 beta4.
```

Original comment by `dean.edw...@gmail.com` on 30 Apr 2010 at 9:14
- Changed state: **Fixed**
